src/main/java/sirius/biz/storage/layer3/uplink/util/RelativePath.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.layer3.uplink.util;
+
+import sirius.biz.storage.layer3.MutableVirtualFile;
+import sirius.kernel.commons.Strings;
+
+/**
+ * Wraps a path to be attached to a {@link MutableVirtualFile} when providing an uplink.
+ */
+public class RelativePath {
```

Comment:
What is the benefit over using https://docs.oracle.com/javase/8/docs/api/java/io/File.html#File-java.lang.String-java.lang.String-

Reply:
+1

Reply:
that this simply wraps a string describing a path on a remote server which doesn't share any api with a File in the local file system

Reply:
As the JavaDoc states: An abstract representation of file and directory pathnames.
A java.io.File object does not necessarily means a real file in a local file system.

Reply:
Or a Path and its [resolve](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#resolve(java.nio.file.Path)) method could be used